docs/components/abbr.md

```diff
@@ -0,0 +1,48 @@
+# matestack core component: abbr
+
+Show [specs](../../spec/usage/components/abbr_spec.rb)
+
+The HTML `abbr` tag implemented in ruby.
+
+## Parameters
+
+This component expects 1 required param, 2 optional configuration params and either yield content or display what gets passed to the `text` configuration param.
+
+#### # title
```

Comment:
Maybe add `title - required` to make clear it's a required param? => we should standardize this soon